apps/ejabberd/src/mod_muc_light_commands.erl

```diff
@@ -115,6 +116,20 @@ commands() ->
         {from, binary},
         {body, binary}
        ]},
+      {result, ok}],
+
+     [{name, delete_room},
+      {category, <<"muc-lights">>},
+      {subcategory, <<"messages">>},
+      {desc, <<"Delete a MUC room.">>},
```

Comment:
Delete a MUC **Light** room.

Reply:
Description message is now updated Piotr.